ospd_openvas/wrapper.py

```diff
@@ -202,10 +201,9 @@ def __init__(self, certfile, keyfile, cafile):
             self.add_scanner_param(name, param)
 
         if openvas_db.db_init() is False:
-            self.add_scan_error(
-                scan_id, host=target,
-                value='OpenVAS Redis Error: Not possible to find db_connection.')
-            return 2
+            logger.error('OpenVAS Redis Error: Not possible '
+                         'to find db_connection.')
+            exit()
```

Comment:
I do not think exiting when `__init__` encounters a problem is a good solution. Wouldn't it be better to raise an exception?